client/post-editor/editor-revisions-list/header.jsx

```diff
@@ -7,15 +7,48 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
 
-const EditorRevisionsListHeader = ( { numRevisions, translate } ) => {
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const EditorRevisionsListHeader = ( {
+	numRevisions,
+	translate,
+	selectNextRevision,
+	selectPreviousRevision,
+} ) => {
 	return (
-		<div className="editor-revisions-list__header">
-			{ !! numRevisions &&
-				translate( '%(revisions)d revision', '%(revisions)d revisions', {
-					count: numRevisions,
-					args: { revisions: numRevisions },
-				} ) }
+		<div>
```

Comment:
Is it possible to have the `ButtonGroup` (mentioned below) inside the `editor-revisions-list__header` wrapper div here without the extra wrapper?

If not, can the `ButtonGroup` get hoisted out to the parent component?